redundant-bmc/src/redundancy.cpp

```diff
@@ -143,6 +143,13 @@ FailoversNotAllowedReasons getFailoversNotAllowedReasons(const Input& input)
         return reasons;
     }
 
+    if (input.failoverInProgress)
```

Comment:
Commit message:

nit: typo `calcFailoversAllowedDuringFailover`

Reply:
thanks, fixed just that with the latest push.